docker-compose.yaml

```diff
@@ -44,21 +49,51 @@ services:
     image: ghcr.io/darpa-askem/data-service:latest
     environment:
       DATASET_STORAGE_BASE_URL: s3://datasets/
+      STORAGE_HOST: http://minio:9000
```

Comment:
minio should be gone now with everything going through s3

Reply:
This is for matt's docker-compose.  It's fine.

Reply:
orchestration uses s3, but docker-compose doesn't. I don't have your S3 keys.
For now, at least, keeping minio in the docker compose.